app/helpers/tags_helper.rb

```diff
@@ -201,18 +190,19 @@ def sub_tag_tree(tag)
   def blurb_tag_block(item, tag_groups=nil)
     item_class = item.class.to_s.underscore
     tag_groups ||= item.tag_groups
-    categories = ['Warning', 'Relationship', 'Character', 'Freeform']
+    categories = ['ArchiveWarning', 'Relationship', 'Character', 'Freeform']
     tag_block = ""
 
     categories.each do |category|
       if tags = tag_groups[category]
         unless tags.empty?
-          class_name = category.downcase.pluralize
+          class_name = category == "ArchiveWarning" ? "warnings" : category.downcase.pluralize
+
           if (class_name == "warnings" && hide_warnings?(item)) || (class_name == "freeforms" && hide_freeform?(item))
             open_tags = "<li class='#{class_name}' id='#{item_class}_#{item.id}_category_#{class_name}'><strong>"
             close_tags = "</strong></li>"
             tag_block <<  open_tags + show_hidden_tags_link(item, class_name) + close_tags
-          elsif class_name == "warnings"
+          elsif class_name == "archive_warnings"
```

Comment:
Looking at line 199, I think this should still be `elsif class_name == "warnings"`